cli/popper/gha.py

```diff
@@ -537,18 +543,27 @@ def __init__(self, action, workspace, env, dry):
         self.cwd = os.getcwd()
 
     def run(self, reuse=False):
-        cmd = self.action.get('runs', ['entrypoint.sh'])
-        cmd[0] = os.path.join('./', cmd[0])
-        cmd.extend(self.action.get('args', []))
-
         cwd = self.cwd
-        if not self.dry_run:
-            if 'repo_dir' in self.action:
-                os.chdir(self.action['repo_dir'])
-                cmd[0] = os.path.join(self.action['repo_dir'], cmd[0])
-            else:
-                os.chdir(os.path.join(cwd, self.action['uses']))
-                cmd[0] = os.path.join(cwd, self.action['uses'], cmd[0])
+        if self.action['uses'] == 'sh':
+            cmd = self.action.get('runs')
+            cmd.extend(self.action.get('args', []))
+
+            if not self.dry_run:
+                os.chdir(cwd)
```

Comment:
depending on where `popper run` is executed, `cwd` might not be the root of the project folder. So we can instead run `pu.get_git_project_root()` here instead.

Reply:
@ivotron Yes, I was only thinking about `main.workflow` in the root.